src/components/dls/ModalNew/Modal.module.scss

```diff
@@ -0,0 +1,87 @@
+@keyframes overlayShow {
```

Comment:
There is an existing keyframes util inside `_utility.scss` which we should use to support different browsers.